### PR TITLE
fix(ps/builds/config): move consistently failing tests to pending

### PR DESCRIPTION
**Pull request by sgoings**

### CODE REVIEW

File: tests/ps_test.go [hunk 1]
```diff
@@ -88,7 +88,7 @@ var _ = Describe("Processes", func() {
 			Entry("scales to 3", 3, 200),
 			Entry("scales to 0", 0, 502),
 			Entry("scales to 5", 5, 200),
-			Entry("scales to 0", 0, 502),
+			PEntry("scales to 0", 0, 502),
 			Entry("scales to 1", 1, 200),
```
Comment (vdice) [inline tests/ps_test.go]: We'll need to `P` this line as well until fix supplied for related issue https://github.com/deis/workflow-e2e/issues/69

Comment (sgoings) [inline tests/ps_test.go]: Stop P-ing on my lines, man. :laughing:

File: tests/ps_test.go [hunk 2]
```diff
@@ -61,7 +61,7 @@ var _ = Describe("Processes", func() {
 			})
 		})
 
-		DescribeTable("can scale up and down",
+		PDescribeTable("can scale up and down",
```
Comment (vdice) [inline tests/ps_test.go]: If this is marked pending, will the associated regular/non-pending `Entry` lines below run?

Comment (sgoings) [inline tests/ps_test.go]: It removes the whole table from the set of tests we run.

**Comment by vdice** [comment]
LGTV!

**Comment by sgoings** [comment]
```
Ran 101 of 114 Specs in 827.586 seconds
SUCCESS! -- 101 Passed | 0 Failed | 13 Pending | 0 Skipped --- PASS: TestTests (827.59s)
PASS
ok  	github.com/deis/workflow-e2e/tests	827.842s
```

**Pull request merged**